scripts/pwm_openmpw/setup-flow.sh

```diff
@@ -0,0 +1,4 @@
+# copy config files to design dir
+sed -i '/::env(DESIGN_IS_CORE)/d' ${design_dir}/openlane/pwm/config.tcl
+sed -i '/::env(GLB_RT_MAXLAYER)/d' ${design_dir}/openlane/pwm/config.tcl
+sed -i '/::env(BASE_SDC_FILE)/d' ${design_dir}/openlane/pwm/config.tcl
```

Comment:
For this design, you removed `BASE_SDC_FILE` but did not add a new one on the `config.tcl`. On other designs, you did add. Any reason for this to be different?